crates/common/validator/src/validator.rs

```diff
@@ -269,6 +421,39 @@ impl ValidatorService {
                     .publish_blinded_block(BroadcastValidation::Gossip, signed_blinded_block)
                     .await?;
             }
+        };
+
+        Ok(())
+    }
+
+    pub async fn prepare_sync_infos(&mut self, slot: u64) -> anyhow::Result<()> {
+        self.sync_normal_infos.clear();
+        self.sync_aggregator_infos.clear();
+
+        for duty in &self.sync_committee_duties {
+            if let Some(keystore) = self.validator_index_to_keystore.get(&duty.validator_index) {
+                for &committee_index in &duty.validator_sync_committee_indices {
```

Comment:
```suggestion
        for duty in &self.sync_committee_duties {
            let Some(keystore) = self.validator_index_to_keystore.get(&duty.validator_index) else {
                continue;
            }
                for &committee_index in &duty.validator_sync_committee_indices {
```

it we be nice to make this code a little less indented